### PR TITLE
feat: add adoption page - virtual adoption steps

### PR DESCRIPTION
**Pull request by paulgrym**
## Jira issue

JIRA issue: [FSU-299](https://klau.atlassian.net/browse/FSU-299)

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Review by Ebmili — APPROVED** [review]
Well done :)

File: packages/frontend/app/templates/Adoption/components/AdoptionRefusal/AdoptionRefusal.jsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const AdoptionRefusal = () => (
 
         <Styled.Features>
           {features.map(({ title, description, icon }) => (
-            <Feature key="title" heading={title} icon={icon} color="gray">
+            <Feature key={title} heading={title} icon={icon} color="gray">
```
Comment (im-klau) [inline packages/frontend/app/templates/Adoption/components/AdoptionRefusal/AdoptionRefusal.jsx:57]: 😅 🙈

**Review by im-klau — APPROVED** [review]
🐶 🎉

File: packages/frontend/app/templates/Adoption/components/VirtualAdoptionSteps/VirtualAdoptionSteps.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            >
+              <Typography variant="bodyTitle">{description}</Typography>
+            </Card>
+          ))}
+        </Styled.CardsContainer>
+      </Styled.BannerContent>
+      <Styled.ScrollContainer>
+        <Typography variant="h3">Sprawdź, kto czeka na opiekuna</Typography>
+        <ScrollButton
+          sectionId="wirtualy-przyjaciel"
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Adoption/components/VirtualAdoptionSteps/VirtualAdoptionSteps.jsx:61]:
```suggestion
          sectionId="wirtualny-przyjaciel"
```

File: packages/frontend/app/templates/Adoption/components/VirtualAdoptionSteps/VirtualAdoptionSteps.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      "Wyślij informację za pomocą krótkiego formularza na stronie, żebyśmy mogli znalezć Twoją wpłatę i informować o stanie pupila.",
+  },
+]
+
+const VirtualAdoptionSteps = () => (
+  <Container as="section">
+    <Styled.Content>
+      <Styled.BannerContainer>
+        <Styled.Banner
+          heading="Co zrobić, aby stać się  wirtualnym opiekunem?"
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Adoption/components/VirtualAdoptionSteps/VirtualAdoptionSteps.jsx:34]:
```suggestion
          heading="Co zrobić, aby stać się wirtualnym opiekunem?"
```

File: packages/frontend/app/templates/Adoption/components/ChooseYourVirtualPet/ChooseYourVirtualPet.jsx [hunk 4]
```diff
@@ -36,7 +36,7 @@ const ChooseYourVirtualPet = ({ data }) => {
   }
 
   return (
-    <Container>
+    <Container id="wirtualy-przyjaciel">
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Adoption/components/ChooseYourVirtualPet/ChooseYourVirtualPet.jsx:39]:
```suggestion
    <Container id="wirtualny-przyjaciel">
```

**Review by AmandaKr — APPROVED** [review]

**Pull request merged**